app/assets/stylesheets/views/_consultation.scss

```diff
@@ -0,0 +1,67 @@
+.consultation {
+  @include sidebar-with-body;
+  @include history-notice;
+  @include withdrawal-notice;
+
+  .section-title {
+    @include bold-27;
+    margin-top: $gutter;
+    margin-bottom: $gutter-half;
+
+    @include media(tablet) {
+      margin-top: 0;
+      margin-bottom: $gutter;
+    }
+  }
+
+  .consultation-notice {
```

Comment:
This shouldn't need to be nested within `.consultation` given the `.consultation-` prefix?